src/test/java/core/basesyntax/service/impl/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import core.basesyntax.strategy.ReturnOperationHandler;
+import core.basesyntax.strategy.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationStrategyImplTest {
+    private OperationStrategy operationStrategy;
+
+    @Before
+    public void setUp() {
+        Map<Operation, OperationHandler> operationHandlerMap = new HashMap<>();
+        operationHandlerMap.put(Operation.BALANCE, new BalanceOperationHandler());
+        operationHandlerMap.put(Operation.PURCHASE, new PurchaseOperationHandler());
+        operationHandlerMap.put(Operation.RETURN, new ReturnOperationHandler());
+        operationHandlerMap.put(Operation.SUPPLY, new SupplyOperationHandler());
+        operationStrategy = new OperationStrategyImpl(operationHandlerMap);
+    }
+
+    @Test
+    public void getOperationHandler_validData_Ok() {
+        Operation operation = Operation.SUPPLY;
+        OperationHandler expected = operationStrategy.getOperationHandler(Operation.SUPPLY);
```

Comment:
```suggestion
        OperationHandler expected = new ReturnOperationHandler();
```
cause now you try to test business logic to get results from business logic